virttest/libvirt_xml/devices/iommu.py

```diff
@@ -0,0 +1,32 @@
+"""
+Support for the iommu device XML
+
+http://libvirt.org/formatdomain.html#elementsDevices
+"""
+
+from virttest.libvirt_xml import accessors
+from virttest.libvirt_xml.devices import base
+
+
+class Iommu(base.UntypedDeviceBase):
+
+    __slots__ = ('model', 'intremap', 'eim')
+
+    def __init__(self, virsh_instance=base.base.virsh):
+        accessors.XMLAttribute(property_name='model',
+                               libvirtxml=self,
+                               parent_xpath='/',
+                               tag_name='iommu',
+                               attribute='model')
+        accessors.XMLAttribute(property_name="intremap",
```

Comment:
I  assume initremap and eim is property under driver element, so suggest  expose as dict
        accessors.XMLElementDict('driver', self, parent_xpath='/',
                                 tag_name='driver')
```
<devices>
  <iommu model='intel'>
    <driver intremap='on'/>
  </iommu>
</devices>
```